### PR TITLE
WIP:Task1#40385

### PR DESCRIPTION
**Pull request by varsha-22**

### CODE REVIEW

**Review by pinku4444 — COMMENTED** [review]
Please remove node_modules folder.
Add node_modules folder in gitignore file.

**Comment by varsha-22** [comment]
> Please remove node_modules folder.
> Add node_modules folder in gitignore file.

node_modules has been removed

File: src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (pinku4444) [inline src/index.jsx:4]: Remove service worker from this file and also remove all useless code.

Comment (pinku4444) [inline src/index.jsx:4]: Remove service worker from this file and also remove all useless code.

**Review by pinku4444 — COMMENTED** [review]
please delete serviceworker.js file from src folder.

File: src/index.jsx (hunk 1 above)
Comment (varsha-22) [reply src/index.jsx:4]: Serviceworker file and all useless code has been removed.

**Comment by varsha-22** [comment]
> please delete serviceworker.js file from src folder.

done

**Comment by varsha-22** [comment]
> please delete serviceworker.js file from src folder.

done.

**Pull request merged**